flow360/component/flow360_params/flow360_fields.py

```diff
@@ -146,8 +146,7 @@ def get_aliases(name) -> List[str]:
     if name in full:
         i = full.index(name)
         return [name, short[i]]
-
-    raise ValueError(f"{name} is not a valid output field name.")
+    return [name, name]
```

Comment:
I think it was better to raise here. I understand the motivation was for unknown field names, whereas this here serves for "known" field names. But I think it should be handled by caller function. Optionally, a parameter can be use `raise_on_not_found=True`

Reply:
Done